v5/v5_with_library/v5_stable.ino

```diff
@@ -1224,7 +1272,7 @@ void wait(unsigned int miliSeconds) {
 
 void setTime(String timeOfMessage) {
   // +CMGL: 1,"REC READ","+923374888420","","23/08/14,17:21:05+20"
-  rtc = fetchDetails(timeOfMessage, "\"23/", "\"", 1);
+  rtc = fetchDetails(timeOfMessage, "\"24/", "\"", 1); // FIXME: Static year
```

Comment:
you can fetch year by myRTC.year then why don't you use that here? @mhamidjamil

Reply:
Sure, Done.